chocolate-pizza-2/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import Header from "./header.js";
+import Footer from "./footer.js";
+import Recipe from "./recipe";
+import logo from './logo.svg';
+import './App.css';
+import chocoPizza from './assets/choco-pizza.png';
+import logoHeader from './assets/logo.png';
+import vanPic from './assets/van-pic.png';
+import printer from './assets/print-icon.png';
+import fb from './assets/fb-icon.png';
+import twit from'./assets/twit-icon.png';
+import gp from'./assets/gp-icon.png';
+import insta from'./assets/insta-icon.png';
+import flic from'./assets/flic-icon.png';
+import pint from'./assets/pint-icon.png';
+import rss from'./assets/rss-icon.png';
+import  mail from './assets/mail-icon.png';
+import smallLogo from './assets/small-logo.png';
+
+function App() {
+  return (
+
+<div>   
+
+    <Header logoHeader={logoHeader} fb={fb} twit={twit} gp={gp} insta={insta} flic={flic} pint={pint} rss={rss} mail={mail}
+     />
+    <main>
```

Comment:
I would make this whole main section broken down into three different components to meet the requirements for the lab. You could do an ingredients component, a title component, and a body component with everything else inside.